src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("the excavator started it`s work");
+
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("the excavator stopped it`s work");
```

Comment:
Similarly, the message in the `stopWork` method should indicate that it is an `Excavator` that has stopped working. Make sure the message is clear and informative.